### PR TITLE
Added components for header

### PR DESCRIPTION
**Pull request by MariethPerez**
Components added whit styles: 
- Header
- Header name
- Header Options
- Header Search

### CODE REVIEW

File: client/src/HeaderName.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  width: 24px;
+  height: 24px;
+  margin: 0 8px;
+  fill: #72767d;
+`;
+
+function HeaderName() {
+  return (
+    <div className={container}>
+      {/* <svg id="btnMenu" onclick="showLeft()"><use xlink:href="#menu"></svg> */}
```
Comment (CesarCachay) [inline client/src/HeaderName.js]: What do you think about changing the commented lines?

File: client/src/HeaderSearch.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  transition: all 0.25s;
+  width: 10vw;
+  :focus {
+    outline: none;
+    width: 15vw;
+  }
+`;
+
+const svg = css`
+  margin: 0;
```
Comment (CesarCachay) [inline client/src/HeaderSearch.js]: Why do you think this line would be necessary? 
I recommend that if the value is "0" we can use another style to improve it :+1:

Comment (luciagirasoles) [reply client/src/HeaderSearch.js]: @CesarCachay I think this margin is to normalize this css?

**Comment by condef5** [comment]
What do you think about changing the title of the pull request?
I recommend use imperative form naming the pull request.

**Comment by MariethPerez** [comment]
Resolve #24

**Pull request merged**